README.md

```diff
@@ -132,22 +132,22 @@ schema:
 `schema.edge` is a **required** parameter that describes the schema information of the inserted edge. Each edge contains the following three properties:
 
 * `name`: The edge's name.
-* `withRanking`: Specifies the `rank` value of the given edge, used to tell different edges share the same edge type and vertices.
+* `withRanking`: Specifies the `rank` value of the given edge, used to tell different edges to share the same edge type and vertices.
 * `props`: Same as the above tag. Please be noted the property order here must be the same with that of the corresponding data in the CSV data file.
 
 Details of all the configurations please refer to [Configuration Reference](docs/configuration-reference.md).
```

Comment:
For details of all the configurations, refer to [Configuration Reference](docs/configuration-reference.md).